portfolio/src/main/webapp/script.js

```diff
@@ -251,3 +252,28 @@ function drawScoreChart() {
 function drawMinChart() {
   drawChart(false);
 }
+
+var markers = [];
+var map;
+
+function createMap() {
```

Comment:
It would be good do explain in the comment what this function is expected to do.

Reply:
done!